sycl/include/CL/sycl/detail/pi_sycl.hpp

```diff
@@ -0,0 +1,12 @@
+//==---------------- pi_sycl.hpp - SYCL wrapper for PI ---------*- C++ -*---==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// ===--------------------------------------------------------------------=== //
+
+#pragma once
+
+#include <CL/sycl/detail/defines.hpp>
+#include <pi/pi.hpp>
```

Comment:
I'm not sure I understand the motivation to have this wrapper.
Do we have a dependency on `defines.hpp` from `pi.hpp`? If so, I think we should remove such tight coupling between plug-in interface and the rest of the sycl runtime library.